src/logic/operate.js

```diff
@@ -0,0 +1,24 @@
+import Big from 'big.js';
+
+const operations = (() => {
+  const operate = (numberOne, numberTwo, operation) => {
+    if (operation === '+') {
+      return Big(numberOne).plus(Big(numberTwo));
+    }
+    if (operation === '-') {
+      return Big(numberOne).minus(Big(numberTwo));
+    }
+    if (operation === '÷') {
+      return Big(numberOne).div(Big(numberTwo));
+    }
```

Comment:
dont forget to manage DIVISION BY ZERO ERROR..........in next milestone, else it will return an error